File: exercises/practice-01.py
#! /usr/bin/env python
# -*- coding: utf-8 -*-
# __author__ = ""
# Date: 2019-07-09

count = 0
for i in [1, 2, 3, 4]:
    for j in [1, 2, 3, 4]:
        for k in [1, 2, 3, 4]:
            if i != j and i != k and j != k:
                count += 1
                print(i, j, k)
print(count)

File: src/exercises/practice-01.go
package main

import "fmt"

// 有1、2、3、4个数字，能组成多少个互不相同且无重复数字的三位数？分别是多少？

func main() {
	var count int
	for i := 1; i <= 4; i++ {
		for j := 1; j <= 4; j++ {
			for k := 1; k <= 4; k++ {
				if (i != j) && (i != k) && (k != j) {
					count += 1
					fmt.Println(i, j, k)
				}
			}
		}
	}
	fmt.Println(count)
}
